azure-pipelines/pull-request-validation/credscan.yml

```diff
@@ -0,0 +1,23 @@
+# File: azure-pipelines\pull-request-validation\credscan.yml
```

Comment:
>credscan [](http://example.com/codeflow?start=48&length=8)

As this does more than credscan, maybe we can rename it to "compliance-checks.yml" or "security-checks.yml"

Reply:
c5a7ded616ea40b9e9b671114777f99b2ad5a175